packages/react-components/src/templates/__tests__/GroupProfilePage.test.tsx

```diff
@@ -9,6 +9,8 @@ const props: ComponentProps<typeof GroupProfilePage> = {
   groupTeamsHref: '#teams',
   lastModifiedDate: '2021-01-01',
   numberOfTeams: 1,
+  pastEventsCount: 0,
+  upcomingEventsCount: 0,
```

Comment:
I kind of feel by injecting the no events component we should be testing the integration of both components in this file. If it was deleted/omitted I don't think any tests would fail